### PR TITLE
extend_linter_integration.md: Update milestones

### PR DESCRIPTION
**Pull request by yash-nisar**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by yash-nisar** [comment]
CC @Makman2

File: data/projects/extend_linter_integration.md [hunk 1]
```diff
@@ -29,24 +29,23 @@ capabilities.
...
-* The applicant has outlined all changes proposed to the linter
-  framework, if needed as a [cEP](https://coala.io/cep).
+* Design the new testing API yielding useful results.
 
 ##### CODING PHASE 1
 
-* At least one linter bear has been merged.
-* At least one improvement to the linter framework is in review state.
-* All of them are properly tested and documented.
+* Modify the testing API to display user friendly results.
```
Comment (Makman2) [inline data/projects/extend_linter_integration.md]: maybe you should add also "add documentation".

File: data/projects/extend_linter_integration.md [hunk 2]
```diff
@@ -29,24 +29,24 @@ capabilities.
...
-  framework, if needed as a [cEP](https://coala.io/cep).
+* Designing the new testing API to yield useful results.
 
 ##### CODING PHASE 1
 
-* At least one linter bear has been merged.
-* At least one improvement to the linter framework is in review state.
-* All of them are properly tested and documented.
+* Modification of the testing API to display user friendly results and add
+corresponding documentation.
```
Comment (Makman2) [inline data/projects/extend_linter_integration.md]: wanna indent it by 2 spaces? ;)

Comment (yash-nisar) [inline data/projects/extend_linter_integration.md]: Why not ;)

**Comment by Makman2** [comment]
@rultor merge

**Comment by Makman2** [comment]
ack 46d8a39

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**